### ISSUE TITLE
Request - Add an option to set the default audio and subtitle language (and subtitles on/off) for an entire series

### ISSUE THREAD

**Issue by zombiezrule** [iwalton3/jellyfin-media-player#749]
**Is your feature request related to a problem? Please describe.**
Watching some series produced in other countries, and the default audio track is set to the English dub. I'd prefer to watch them in the native language with English subtitles, but the language often (not always) changes between episodes.

**Describe the solution you'd like**
An option to set the default audio language and subtitle language/track for a whole series, either through the series' metadata or an option when viewing the season selection screen after selecting the series.

**Describe alternatives you've considered**
A setting that allows the client to remember if an audio and subtitle track was manually selected, and to follow that manual selection for future episodes could also work.

**Additional context**
There's already an option to change the audio and subtitle track for an individual episode when viewing the episode selection screen, extending this to an entire series would be nice.

**Comment by MedAzizTousli** [comment]
I believe that this is a good idea that can help as a workaround for #2883.

**Comment by dmg77** [comment]
I agree to adding a subtitle selection for an entire series would be a great benefit for jellyfish users especially those that use the TV app like I. 

For example, I have a show that I'm watching that has a sign and songs subtitle that is set as default and I have to manually switch it to the full set of subtitles to get any subtitles from actual dialogue. But if I could just send the full subtitles to default from the shows main page that it would save me such trouble.

**Comment by GreenGinkgo** [comment]
Yep! This is also my pet peeve with Plex.

I have this same issue, at least on the web player on v10.10.7. Here's an example where there's a "Default" on the video file, but I would like to set the dialogue to Japanese and the Subtitles to the "Dialogue" version of English instead of the default "Signs & Song". (I selected "Off" in the screencap just so that the bottom selector didn't cover the top one).

<img width="482" height="170" alt="Image" src="https://github.com/user-attachments/assets/165a4b28-fad1-4b8b-b8bc-56a73f21567e" />

If there would be a way to display a subtitle selector at least per-season, that would be amazing.
I'd imagine it would involve scraping the subtitle options from all files in that season, and then present a list of the selectable ones (in case a few eps have subtitle names that differ from the rest, maybe those are an option on the file itself?).

Also maybe the library owner could add a search/regex to the series' metadata that would select whatever subtitle name matches. So in the case above, I might be able to re-define a default to be something like `/dialog.*eng(lish)?/i` ("any mention of "dialog" followed at some point by "eng" or "english", case-insensitive).

**Comment by SantinoMusi** [comment]
I completely agree that this is a fantastic idea

It would also be great to have a setting for "Play original audio track, regardless of language", so that, for example, anime always automatically played in Japanese even if I have English audio as default

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
